runtime/development/src/lib.rs

```diff
@@ -791,8 +791,14 @@ impl pallet_fees::Config for Runtime {
 	type WeightInfo = weights::pallet_fees::SubstrateWeight<Self>;
 }
 
+// Parameterize anchors pallet
+parameter_types! {
+	pub const MaxBound: u32 = 100;
```

Comment:
I think the child root is a `H256`, so 256 bytes. The bound should allow this size if I am not mistaken. WDYT? @branan